test/WebJobs.Extensions.DurableTask.Analyzers.Test/ActivityFunction/FunctionReturnTypeAnalyzerTests.cs

```diff
@@ -41,9 +38,9 @@ public static class HelloSequence
             {
                 var outputs = new List<string>();
 
-                outputs.Add(await context.CallActivityAsync<int>(""E1_SayHello"", ""Tokyo""));
-                outputs.Add(await context.CallActivityAsync<int>(""E1_SayHey"", ""Tokyo""));
-                outputs.Add(await context.CallActivityAsync<int>(""E1_SayHello_Tuple"", (""Seattle"", 4)));
+                await context.CallActivityAsync<int>(""E1_SayHello"", ""Tokyo"");
+                await context.CallActivityAsync<int>(""E1_SayHey"", ""Tokyo"");
+                await context.CallActivityAsync<int>(""E1_SayHello_Tuple"", (""Seattle"", 4));
```

Comment:
Why this change? The C# is no longer valid...

Reply:
Technically it wasn't valid before. I was adding int to a string array. Now I'm just not doing anything with the result. So now it bothers me slightly less.

Reply:
But why change this at all?

Reply:
I changed it to make the C# valid. Also, adding my result to an outputs list isn't part of my test, so I wanted to simplify the test code as much as possible to make my test slightly more straightforward. My analyzer is looking for CallActivityAsync calls.

Reply:
Also consider that these tests are taking this string and running my analyzer on them to see which new warnings/errors appear. I just want to keep them as simple as possible, while having the structure of durable functions customer code.